.github/SECURITY.md

```diff
@@ -29,15 +29,13 @@ to learn more, head to the
 The Apache Software Foundation takes security issues very seriously. Apache
 Airflow specifically offers security features and is responsive to issues
 around its features. If you have any concern around Airflow Security or believe
-you have uncovered a vulnerability, we suggest that you get in touch via the
-e-mail address [security@airflow.apache.org](mailto:security@airflow.apache.org).
+you have uncovered a vulnerability, we suggest that you privately report it by using the [vulnerability reporting form](https://github.com/airflow-s/airflow/security/advisories/new).
```

Comment:
I thought that the other fork was just for testing

Reply:
But yeah, I think it's better to keep everything related to the security there.

Reply:
In order to test things I figured we need people to use that new way of reporting, and redirecting them would be a good start.

We can also redirect when we receive emails at security@airflow.apache.org/private@airflow.apache.org